memegen/templates/image.html

```diff
@@ -1,27 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
+  <head>
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@jasich">
```

Comment:
Let's change this to `@memegen_link`. Maybe that account can become a bot later that tweets popular memes. 😄